src/EA.Weee.RequestHandlers/Admin/AddAatfRequestHandler.cs

```diff
@@ -41,6 +42,15 @@ public async Task<bool> HandleAsync(AddAatf message)
 
             Operator @operator = await dataAccess.GetSingleByExpression<Operator>(new OperatorByOrganisationIdSpecification(message.OrganisationId));
 
+            if (@operator == null)
```

Comment:
Can we add a unit test to check if the operator is null a operator is added to the context and the aatf used this operator

Reply:
Done